juniper_official/interface/check-ifl-state.rule

```diff
@@ -111,6 +126,18 @@ healthbot {
                 }
                 type float;
                 description "Utilization out percentage which derived based on mbps and interface out bw";
+                plots out-traffic-util-plots {
+                    unit percentage;
+                    triggers sub-interface-output-traffic;
+                    thresholds high-threshold {
+                        field-name "$high-threshold";
+                        severity critical;
```

Comment:
Can you add plots to in and out mbps a well. Do not add thresholds for those plots

Reply:
plots input-traffic-mbps {
                    unit mbps;
                    triggers sub-interface-input-traffic;
                }

        plots output-traffic-mbps {
                    unit mbps;
                    triggers sub-interface-output-traffic;
                }